src/health_probes.ts

```diff
@@ -0,0 +1,32 @@
+import { createServer, Server } from 'http';
+
+/* 
+ * This is a very simplified readiness probe that responds to any url with a status code
+ * Can be further extended to provide metrics and 
+ */
+
+export class Healthprobe {
+
```

Comment:
```suggestion
```

Reply:
you generally have a lot of newlines that could go away and make the code look a bit scattered. Setting up a strict formatter will help.